client/client_test.go

```diff
@@ -918,6 +930,77 @@ func testListTarget(t *testing.T, rootType string) {
 	assert.Equal(t, currentTarget, newCurrentTarget, "current target does not match")
 }
 
+func testListTargetWithDelegates(t *testing.T, rootType string) {
```

Comment:
I might just be missing it, but I don't see where this function is called.  Does it need to be added to `TestListTarget`? (or a new test case?)